frontend/src/components/Toggle.jsx

```diff
@@ -22,7 +24,7 @@ export default class Toggle extends Component {
     render() {
         return (
             <a
-                className={cx("Toggle", "no-decoration", { selected: this.props.value }) + " " + (this.props.className||"")}
+                className={cx(styles.toggle, "no-decoration", { [styles.selected]: this.props.value }) + " " + (this.props.className||"")}
```

Comment:
@tlrobinson would it make more sense here to just fold the css from Toggle.css into this react component?  seems a little strange to go to the effort to have the css in a separate file if its use is scoped to just a react component.